pilot/pkg/networking/plugin/authz/rbac.go

```diff
@@ -71,6 +72,11 @@ const (
 	attrRequestClaims      = "request.auth.claims"         // claim name is surrounded by brackets, e.g. "request.auth.claims[iss]".
 	attrRequestClaimGroups = "request.auth.claims[groups]" // groups claim.
 
+	// reserved string values in names and not_names in ServiceRoleBinding.
+	// This prevents ambiguity when the user defines "*" for names or not_names.
+	allUsers              = "allUsers"              // Allow all users, both authenticated and unauthenticated.
```

Comment:
Make sure to document this in istio.io.

Reply:
I added a comment to the Authz-v2 doc. As for the doc on istio.io website, it doesn't seem like https://github.com/istio/istio.io has a release-1.2 branch yet?

Reply:
Got it. Thanks!